VUDU/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+      "name": "Autumn",
+      "id": "374245143655612428"
+    },
+    "url": "www.vudu.com",
+    "description": {
+      "en": "This extension adds rich presence support to VUDU, a movie site."
+    },
+    "service": "VUDU",
+    "version": "1.0.0",
+    "logo": "https://www.vudu.com/content/images/logo.svg",
+    "thumbnail": "https://cdn.discordapp.com/attachments/377284586326982666/697982989048741928/unknown.png",
+    "color": "#236cba",
+    "tags": [
+      "movies"
+    ],
+    "category": "videos",
+    "iframe": false

```

Comment:
iframe false isnt needed since iframe is optional

Dont forget to remove the comma behind category when you remove it